recipes/godot_headers/all/conandata.yml

```diff
@@ -0,0 +1,4 @@
+sources:
+  "cci.3.2-20200611":
+    url: "https://github.com/godotengine/godot_headers/archive/ddf67cc7b8274c5fb77a71c828bab2991f1ee12a.zip"
```

Comment:
This is the commit of 2020/01/30: https://github.com/godotengine/godot_headers/commit/ddf67cc7b8274c5fb77a71c828bab2991f1ee12a

Reply:
Good catch! I will change the version accordingly, since godotengine/godot_headers@ddf67cc is the one referenced in godot-cpp.